src/apps/concepts/components/AddConceptToDictionaryIcon.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+    Fab,
+    Tooltip
+} from "@material-ui/core";
+import {APIConcept} from "../types";
+
+import {Add as AddIcon} from "@material-ui/icons";
+
+
+
+interface AddConceptToDictionaryIconProps {
+    dictionaryToAddTo:string;
+    canAddConcept:string;
```

Comment:
Shouldn't this be a boolean?

Reply:
yes but as a query string Parameter it is taking 'String' as the default data type

Reply:
but @ibacher  as default datatype of query param, it is String